web/ui/navbar/index.svelte

```diff
@@ -49,28 +52,43 @@
 
 <nav class="navbar">
   <div class="navbar_main" aria-orientation="horizontal" role="menu">
-    {#if $isRefreshing}
+    <div class={`navbar_back-button ${!$secondStep ? 'is-hidden' : ''}`}>
       <NavbarItem
-        name={$t.refresh}
-        current={$router.route === 'refresh'}
-        hotkey="r"
-        href={getURL('refresh')}
-        small
-      >
-        <NavbarProgress value={$refreshProgress} />
-      </NavbarItem>
-    {:else}
-      <NavbarItem
-        name={$t.refresh}
-        current={$router.route === 'refresh'}
-        hotkey="r"
-        icon={mdiRefresh}
+        name={$t.back}
+        current={true}
+        hotkey="Escape"
```

Comment:
There 2 problems with `Esc` handling right now:

1. Button is too small for `Escape` hotkey preview
![Captura desde 2024-05-13 12-54-35](https://github.com/hplush/slowreader/assets/19343/a3dcfae1-65b5-4e49-ace9-5fc68c417e3d)
2. The `Esc` handling works unpredicted on desktop.

Case 1:
1. Open desktop mode
2. Add `github.blog` as Slow
3. Open Slow
4. Open GitHub
5. Click on Open post
6. Press Esc → Post will be closed (Great), focus will be returned to posts list (not so good, since we had no focus, but OK)
7. Press Esc again → focus can’t be disabled anymore and always return to the posts list (expected behaviour: no focus)

Case 2:
1. Open add
2. Enter `github.blog`
3. Press Enter → focus moves to the links list
4. Press Enter → focus moved to the side bar
5. Press Esc → focus return to the field, not to the list as expected

Case 3:

1. Add `github.blog`
2. Open By categories
3. Click on GitHub → side panel will be open
4. Press Esc → side panel will not be closed (until I press Esc again)

I assume there is Esc conflict with `keyux` jumps. Let's remove `hotkey="Escape"` and do right Escape handling as separated task.